### PR TITLE
feat: allow changes to banners using Growthbook

### PR DESCRIPTION
**Pull request by VB-at-Bis**
Implements #722

### CODE REVIEW

File: app/pages/index.tsx [hunk 1]
```diff
@@ -74,49 +63,39 @@
...
+          <>
           Refer to{' '}
           <Link href="https://www.gov.bc.ca/connectingcommunitiesbc">
             program details
           </Link>{' '}
           for the application materials and full information about the
           Connecting Communities British Columbia (CCBC) program.
-          <StyledCallout>{intakeCalloutChildren}</StyledCallout>
+          {intakeCalloutChildren}
+          </>
```
[the remark this answers is not in the document]
Comment (VB-at-Bis) [reply app/pages/index.tsx]: latest code in this branch does not have `intakeCalloutChildren` anymore.. but code scanner did not mark it as `fixed` for some reason

File: app/pages/dashboard.tsx [hunk 2]
```diff
@@ -82,39 +81,23 @@ const Dashboard = ({
...
-                <Link
-                  href="https://www2.gov.bc.ca/gov/content/governments/connectivity-in-bc/20601/20601-63737"
-                  target="_blank"
-                  rel="noopener noreferrer"
-                >
-                  program webpage
-                </Link>{' '}
-                for updates.
-              </p>
-            </div>
```
Comment (matthieu-foucault) [inline app/pages/dashboard.tsx:117]: This should not be deleted. The dynamic alert should be added above, but should not replace this

File: app/pages/index.tsx [hunk 3]
```diff
@@ -74,49 +62,31 @@ const Home = ({
...
+  const openIntakeBanner = useFeature('open_intake_alert').value || {};
+  const closedIntakeBanner = useFeature('closed_intake_alert').value || {}; 
 
   return (
     <Layout session={session} title="Connecting Communities BC">
       <div>
         <h1>Welcome</h1>
-        {!openIntake && (
-          <IntakeAlert openTimestamp={nextIntake?.openTimestamp} />
-        )}
```
Comment (matthieu-foucault) [inline app/pages/index.tsx:111]: This is the only part of this file that should be updated, the callout below should remain, and the dynamic alerts should be displayed here

**Review by matthieu-foucault — CHANGES_REQUESTED** [review]

File: app/pages/dashboard.tsx (hunk 2 above)
Comment (VB-at-Bis) [reply app/pages/dashboard.tsx]: changes rolled back

File: app/pages/index.tsx (hunk 3 above)
Comment (VB-at-Bis) [reply app/pages/index.tsx:111]: done

File: app/components/DynamicAlert.tsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+      <>
+        <br />
+          Please check the{' '}
+          <Link href="https://www.gov.bc.ca/connectingcommunitiesbc"
+            target="_blank"
+            rel="noopener noreferrer">
+            program webpage
+          </Link>{' '}
+          for updates.
+      </>}
```
Comment (matthieu-foucault) [inline app/components/DynamicAlert.tsx:52]: I don't think we need to show the link in the alert

File: app/pages/dashboard.tsx [hunk 5]
```diff
@@ -82,15 +83,24 @@ const Dashboard = ({
...
-          <h1>Dashboard</h1>
+          {openIntake && (
+            <DynamicAlert dateTimestamp={closeTimestamp} text={openIntakeBanner.text}
+            variant={openIntakeBanner.variant} includeLink={false} displayOpenDate={false} />
+          )}
           {!openIntake && (
-            <IntakeAlert openTimestamp={nextIntake?.openTimestamp} />
+            <DynamicAlert dateTimestamp={nextIntake?.openTimestamp} text={closedIntakeBanner.text}
+            variant={closedIntakeBanner.variant} includeLink 
+            displayOpenDate = {closedIntakeBanner.displayOpenDate}/>
```
Comment (matthieu-foucault) [inline app/pages/dashboard.tsx:101]: This doesn't look properly formatted. Can you use Prettier to format your files? (VSCode can do that on save with the prettier plugin). Alternatively, the pre-commit hook will ensure your files are formatted

Comment (VB-at-Bis) [reply app/pages/dashboard.tsx]: Can you please provide .prettierrc with your formatting rules?
I tried with 
```{
  "semi": true,
  "tabWidth": 2,
  "printWidth": 100,
  "singleQuote": true,
  "trailingComma": "none",
  "jsxBracketSameLine": true
}
 
but it produced too many changes

Comment (matthieu-foucault) [reply app/pages/dashboard.tsx]: we have a `.prettierrc` at the root of this repo

File: app/components/DynamicAlert.tsx (hunk 4 above)
Comment (VB-at-Bis) [reply app/components/DynamicAlert.tsx]: Removed

File: app/pages/dashboard.tsx (hunk 5 above)
Comment (VB-at-Bis) [reply app/pages/dashboard.tsx]: updated two files only

File: app/pages/index.tsx [hunk 6]
```diff
@@ -102,21 +102,39 @@ const Home = ({
...
-          <StyledCallout>{intakeCalloutChildren}</StyledCallout>
+          <>
+            Refer to{' '}
+            <Link href="https://www.gov.bc.ca/connectingcommunitiesbc">
+              program details
+            </Link>{' '}
+            for the application materials and full information about the
+            Connecting Communities British Columbia (CCBC) program.
+            <StyledCallout>{intakeCalloutChildren}</StyledCallout>
+          </>
```
Comment (marcellmueller) [inline app/pages/index.tsx:137]:
```suggestion
            Refer to{' '}
            <Link href="https://www.gov.bc.ca/connectingcommunitiesbc">
              program details
            </Link>{' '}
            for the application materials and full information about the
            Connecting Communities British Columbia (CCBC) program.
            <StyledCallout>{intakeCalloutChildren}</StyledCallout>
```

**Review by marcellmueller — APPROVED** [review]
Looks good, nice job! Can you just remove useless fragment by the callout before you merge please?

File: app/pages/index.tsx (hunk 6 above)
Comment (VB-at-Bis) [reply app/pages/index.tsx]: Done

**Review by matthieu-foucault — APPROVED** [review]

**Pull request merged**